cmd/tldr/main.go

```diff
@@ -156,6 +160,28 @@ func updatePages() {
 	}
 }
 
+func printHistory() {
+	repository, err := cache.NewRepository(remoteURL, ttl)
+	if err != nil {
+		log.Fatalf("ERROR: creating cache repository: %s", err)
+	}
+
+	history, err := repository.LoadHistory()
+	if err != nil {
+		log.Fatalf("ERROR: error loading history: %s", err)
+	}
+
+	hisLen := len(*history)
+	if hisLen == 0 {
+		fmt.Println("No history is available yet")
+	} else { //default print last 10.
+		for i := 1; i <= 10; i++ {
```

Comment:
This results in a panic (index out of range) if you would not have as much different pages in your history.

```suggestion
		for i := 1; i <= hisLen; i++ {
```

Reply:
thanks. good point.